pkg/policy/l4.go

```diff
@@ -184,6 +181,43 @@ func CreateL4Filter(fromEndpoints []api.EndpointSelector, rule api.PortRule, por
 	return l4
 }
 
+// CreateL4EgressFilter creates an L4Filter for the specified api.PortProtocol in
+// the egress direction for a particular protocol.
+// This L4Filter will only apply to endpoints covered by `fromEndpoints`.
+// `rule` allows a series of L7 rules to be associated with this L4Filter.
+func CreateL4EgressFilter(toEndpoints []api.EndpointSelector, rule api.PortRule, port api.PortProtocol,
```

Comment:
AFAICT this is exactly the same as `CreateL4IngressFilter` except for one boolean (l4.Ingress). We can share this code and pass an additional bool.

`func CreateL4Filter ... ingress bool...`

```
func CreateL4EgressFilter ... {
    return CreateL4Filter(..., false)
}
```

```
func CreateL4IngressFilter ... {
    return CreateL4Filter(..., true)
}
```

Reply:
perhaps reuse the direction type from policymap? or make an enum in this package. boolean literals are a little opaque.